### PR TITLE
Convert TermsOfServiceRepository to Kotlin

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
Towards #532 

Tested for regressions by running the app locally and verified that terms of service is displayed normally.

@JSunde PTAL?

### CODE REVIEW

File: gnd/src/main/java/com/google/android/gnd/repository/TermsOfServiceRepository.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import io.reactivex.Maybe
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+
+private const val LOAD_REMOTE_PROJECT_TERMS_OF_SERVICE_TIMEOUT_SECS: Long = 30
+
+class TermsOfServiceRepository @Inject constructor(
+    private val remoteDataStore: RemoteDataStore, private val localValueStore: LocalValueStore) {
+
+    val termsOfService: @Cold Maybe<TermsOfService>
```
Comment (scolsen) [inline gnd/src/main/java/com/google/android/gnd/repository/TermsOfServiceRepository.kt:32]: you can hide the setter for this if we don't call it elsewhere:

`private set`

Comment (scolsen) [reply gnd/src/main/java/com/google/android/gnd/repository/TermsOfServiceRepository.kt:32]: oops my bad, this is a val so there is no setter

**Comment by scolsen** [comment]
lgtm

**Comment by shobhitagarwal1612** [comment]
Thanks Scott!

**Pull request merged**